### PR TITLE
Updated to add the slack alert channel details and in components page…

### PR DESCRIPTION
**Pull request by Sandhya1874**
… added github enforce admin branch protection

### CODE REVIEW

File: server/views/pages/component.njk [hunk 1]
```diff
@@ -130,6 +130,39 @@
...
+        <tr>
+        <th>Non-Production Alerts Slack Channel</th>
+          <td>
+          {% if component.alerts_nonprod_slack_channel | length %}
+            <a class="govuk-link--no-visited-state" href="slack://channel?team=T02DYEB3A&id={{ component.alerts_nonprod_slack_channel }}">
+            <image src="/assets/images/slack_favicon_32.png" width="32" height="32"></image>
+            {% if component.alerts_nonprod_slack_channel | length %}
+            {{ component.alerts_nonprod_slack_channel }}
+            {% else %}
+            Link
```
Comment (psoleckimoj) [inline server/views/pages/component.njk:159]: What does `Link` mean?

Comment (psoleckimoj) [reply server/views/pages/component.njk:159]: Surely this must exist for you to get to this line?

Comment (psoleckimoj) [reply server/views/pages/component.njk:159]: You've done the same `if` statement inside itself?

Comment (Sandhya1874) [reply server/views/pages/component.njk]: Resolved this

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**